src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/config/ExceptionHandler.kt

```diff
@@ -103,10 +104,11 @@ object EmailInvalid : MagicLinkRequestErrorCodes("INVALID_EMAIL", "Enter an emai
 object EmailInvalidCjsm : MagicLinkRequestErrorCodes("INVALID_CJSM_EMAIL", "Enter an email address which ends with 'cjsm.net'")
 
 sealed class CheckBarcodeErrorCodes(code: String, userMessage: String) : StandardErrorCodes(code, userMessage)
-class Duplicate(val scannedDate: Instant, val scannedLocation: String) : CheckBarcodeErrorCodes("DUPLICATE", "Someone scanned this barcode ${scannedDate.format()} at $scannedLocation. It may be an illegal copy.")
+class Duplicate(val scannedDate: Instant, val scannedLocation: String) : CheckBarcodeErrorCodes("DUPLICATE", "Someone scanned this barcode ${scannedDate.formatAtTimeOnDate()} at $scannedLocation. It may be an illegal copy.")
+class Expired(val createdDate: Instant, val barcodeExpiry: Duration) : CheckBarcodeErrorCodes("EXPIRED", "This barcode was created ${createdDate.ageInDays()}, ${createdDate.formatOnDate()}.")
```

Comment:
Would it be better for this Expired class to have the actual expiry date, rather than an expiry duration? As it stands it means consumers of the service (the front end express app) have to work out the expiry date